docs/specifications/tests/Methods.md

```diff
@@ -1,9 +1,20 @@
-# Methods common to Test Specifications
+# Methods common to Test Specifications (version 1)
```

Comment:
Test Specifications -> Test Case specifications

Reply:
Fixed.

In general, I have not updated the version 1 besides the reference to version 2.